### ISSUE TITLE
Simplify Build Process & Usage

### ISSUE THREAD

**Issue by jinay1991** [euspectre/kernel-strider#7]
Need to simplify the Build process as it takes lots of time in just CMake.

Usage requires lots of Steps to perform in order to identify Race Condition and can be reduced to either one C/C++ based app/library

**Comment by euspectre** [comment]
Yes, the process is more complicated that I would like it to be.

Still, I have no plans to change it in the current version of KernelStrider, the reasons are the same as for KEDR 0.x. That is, KernelStrider will also become a part of KEDR 1.x, where the build process is going to be quite different. Right now, KernelStrider is in "maintenance mode", no big changes are to be introduced here.
